### ISSUE TITLE
there's some simple tutorial?

### ISSUE THREAD

**Issue by elviskudo** [clevertech/YiiBackbone#11]
there is some simple tutorial for develop this system?

coz i got the little problem here, about understand flow from backbone to yii, its about url settings in each js file, about relational

what first i have to do to create some module like forum, in modules yii then integrate with backbone

thanks again,
i like backbone and i want to create with it :+1:

**Comment by itsazzad** [comment]
Yes. It would be great if there be a more elaborate documentation/tutorial for this project

**Comment by cthorne66** [comment]
Take a look at the pull request I am proposing for this project.  Perhaps you can gauge if it's helpful or not.

Thanks
